### ISSUE TITLE
Interface Resolver should allow defining field resolvers alongside __resolveType

### ISSUE THREAD

**Issue by Akryum** [dotansimha/graphql-code-generator#1454]
**Describe the bug**
Currently the type generated for the resolvers of an interface contains only `__resolveType` despite the interface having resolvable fields.

It's possible with Apollo Server to write resolvers for an interface so that all implementations share those:

```js
const schema = makeExecutableSchema({
  typeDefs,
  resolvers,
  inheritResolversFromInterfaces: true,
})
```

**To Reproduce**

```graphql
interface Identity {
  id: ID!
  avatar: Avatar
}

type Avatar {
  url: String!
}
```

```js
const resolvers: IResolvers = {
  Identity: {
    __resolveType: (identity: any, args, ctx) => {
      if (identity.organization_id) {
        return 'TeamMember'
      }
      return null
    },
    avatar: (identity, args, ctx) => {
      return {
        url: `${process.env.NODE_APP_CDN}/uploads/identity/avatar/${identity.avatar}`,
      }
    },
  },
}
```

Which throws this Typescript error:

```
Type '{ __resolveType: (identity: any, args: Context, ctx: GraphQLResolveInfo) => "TeamMember"; avatar: (identity: any, args: any, ctx: any) => { url: string; }; }' is not assignable to type 'Resolvers'.
  Object literal may only specify known properties, and 'avatar' does not exist in type 'Resolvers'.
```

**Expected behavior**
Have all the correct types on `avatar` resolver.

**Environment:**
 - OS: elementary OS luna (Ubuntu 16.04)
 - Codegen: 0.18.0
 - Node: 11.10.0

**Additional context**
-

**Comment by dotansimha** [comment]
Thank you for reporting it @Akryum ! 
We are currently working on a big refactor for the TypeScript packages, so we'll fix this issue there.

**Comment by dotansimha** [comment]
Fixed in: https://github.com/dotansimha/graphql-code-generator/pull/1483

**Comment by dotansimha** [comment]
Fix available in 1.0.0 👍
